create_package.py

```diff
@@ -248,21 +258,33 @@ def get_client_files_mapping() -> List[Tuple[str, str]]:
 
     """
     # Add client code content to zip
-    client_code_dir: str = os.path.join(CLIENT_ROOT, ADDON_CLIENT_DIR)
-    mapping = [
-        (path, os.path.join(ADDON_CLIENT_DIR, sub_path))
-        for path, sub_path in find_files_in_subdir(client_code_dir)
-    ]
+    if ADDON_CLIENT_DIR:
```

Comment:
This is called only if `ADDON_CLIENT_DIR` is set, if is not set. You can't get client content from `CLIENT_ROOT`.